### ISSUE TITLE
A11Y_Oracle Data Migration extension for Azure Data Studio_data migration oracle_Azure data Base Migration Oracle_Screen Reader: Screen Reader is not announcing the "Connection status" error/success messages after selecting the required fields.

### ISSUE THREAD

**Issue by PadminiGariki** [microsoft/azuredatastudio#24924]
### GitHub Tags:
#A11yWCAG2.1; #A11yTCS; #A11ySev2; #DesktopApp; #Benchmark; #BM-AzureDataStudio-Win32-Feb2022; #Azure Data Studio; #Narrator; #WCAG4.1.3; #Status Message;#STP; #FTP;  #ScreenReader; #Win11; #Win32; #NVDA;

### Environment Details:
Application Name: Oracle Data Migration extension for Azure Data Studio
Download application: Azure Data Studio.
Version:1.45.1(system setup)
Operating System: Window 11 version 22H2 OS build 25267.1000.
Screen Reader: NVDA Version 2023.1(2023.1.0.27913)
### Repro Steps:

1. Open Azure Data Studio application.
2. Install "Data Migration for Oracle"
3. connect to source connection.
4. Tab till new Migration and invoke on it.
5. Tab till step-5 "Azure database Migration oracle" and invoke on it.
6. Now enable narrator.
7. Tab till "Resource group" and select any drop down menu item.
8. Tab till "Azure data migration service" and select any drop down menu item.
9. Now verify that Narrator is not announcing the "Connection status" error/success messages after selecting the required fields.

### Actual Result:
Screen Reader is not announcing the "Connection status" error/success messages after selecting the required fields.

**Note:**
Similar issue is observed with NVDA.
### Expected Result:
Screen Reader should announce the "Connection status" error/success messages after selecting the required fields.
### User Impact:
Screen Reader users will not be able to know the "Connection status" error/success messages. If narrator is not announced.
### Attachments
![2663147_code snippet](https://github.com/microsoft/azuredatastudio/assets/142479541/1d87fcf1-6228-444d-83b1-753097603df2)

https://github.com/microsoft/azuredatastudio/assets/142479541/9ac32193-5cec-4910-b616-44b68371e1aa

https://github.com/microsoft/azuredatastudio/assets/142479541/d546b147-1bae-4314-be0f-e13c9d811c38

**Comment by PadminiGariki** [comment]
Prime bug id: https://msdata.visualstudio.com/Migration%20Assistant/_workitems/edit/2663147

**Comment by PadminiGariki** [comment]
As primary team confirm this is not related to Azure data studio. 
We close this work item.
Thank you!
#Regressed:01-23-24; #Closed;

**Comment by kisantia** [comment]
closing as this was confirmed fixed above.
